primitives/keyring/src/sr25519.rs

```diff
@@ -113,6 +113,33 @@ impl From<Keyring> for sp_runtime::MultiSigner {
 	}
 }
 
+#[derive(Debug)]
+pub struct ParseKeyringError;
```

Comment:
Do we still need these lines here?

Reply:
No but it's good to have it if someone wants to make a CLI and use `--account` instead of the `--alice` and the others. I have the idea that sc-cli is a crate for the user to help build the CLI so it's good to be generic.